packages/react-devtools-core/src/standalone.js

```diff
@@ -242,8 +242,20 @@ function connectToSocket(socket: WebSocket) {
   };
 }
 
-function startServer(port?: number = 8097) {
-  const httpServer = require('http').createServer();
+type ServerOptions = {
+  key?: string,
+  cert?: string,
+};
+
+function startServer(
+  port?: number = 8097,
+  host?: string = 'localhost',
+  secure?: boolean = false,
+  httpsOptions?: ServerOptions,
```

Comment:
@bvaughn - Yep, you are right. 

I thought it would be cleaner to have it explicit but definitely can be removed. Let me know what is more inline with the React standards and I'll change.

Reply:
I don't feel super strongly, but since it would be invalid to specify `httpsOptions` without `secure` *or* to specify `secure` without `httpsOptions` --- I learn towards wanting to just merge them.

Reply:
Done!